test/integration_tests/conftest.py

```diff
@@ -53,6 +58,21 @@ def sample_speech(tmp_path, lang):
     return path
 
 
+@pytest.fixture
+def mixture_source():
+    path = torchaudio.utils.download_asset(f"test-assets/{_MIXTURE_FILE}")
```

Comment:
The use of slash prevents this test from running in Windows. Can you use os.path or pathlib?